pyiron/gpaw/gpaw.py

```diff
@@ -46,15 +46,15 @@ def plane_wave_cutoff(self):
     def plane_wave_cutoff(self, val):
         self.input["encut"] = val
 
-    def get_k_mesh_by_cell(self, cell=None, kpoints_per_angstrom=1):
+    def get_k_mesh_by_cell(self, kpoints_per_reciprocal_angstrom, cell=None):
```

Comment:
Why is this function defined here again? Should not GPAW be derived from the GenericDFT class to have access to this function? Duplicate code is always hard to maintain.